doc/user-guide.md

```diff
@@ -268,7 +266,7 @@ We can now invoke the Write RPC on each `node` in the configuration:
   }
 
   // Invoke Write RPC on all nodes in config
-  for _, node := range allNodesConfig.Nodes() {
+  for _, node := range allNodesConfig {
     respons, err := node.Write(context.Background(), state)
```

Comment:
Typo: respons

Although I think we should use `reply` instead since we use `reply` in other parts of the document.